ui-2/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
please update the readme with some text that explain our experiment

Reply:
sure